src/routes/movies.py

```diff
@@ -1,13 +1,236 @@
-from fastapi import APIRouter, Depends, HTTPException, Query
+from datetime import timedelta, date
+
+from fastapi import APIRouter, Depends, HTTPException, Query, Response, Path
 from sqlalchemy import select, func
-from sqlalchemy.exc import IntegrityError
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.orm import joinedload
 
-from database import get_db, MovieModel
-from database.models import CountryModel, GenreModel, ActorModel, LanguageModel
-
+from src.database import get_db
+from src.database.models import CountryModel, GenreModel, ActorModel, LanguageModel, MovieModel
+from src.schemas.movies import (MovieListResponseSchema,
+                                MovieCreateSchema,
+                                MovieOut,
+                                MovieUpdateSchema,
+                                MovieListItemSchema,
+                                MovieDetailSchema
+                                )
```

Comment:
Place all imports on a new line, so there will be no imports on the same line with `(` and `)` in the future
```suggestion
from src.schemas.movies import (
    MovieListResponseSchema,
    MovieCreateSchema,
    MovieOut,
    MovieUpdateSchema,
    MovieListItemSchema,
    MovieDetailSchema
)
```